ticketutil/bugzilla.py

```diff
@@ -231,6 +234,45 @@ def add_comment(self, comment, **kwargs):
             logging.error("Error adding comment to ticket")
             logging.error(e.args[0])
 
+    def add_attachment(self, data, file_name, summary, **kwargs):
+        """
+        :param data: The content of the attachment which is base64 encoded.
+        :param file_name: The "file name" that will be displayed in the UI for this attachment.
+        :param summary: A short string describing the attachment.
+        :return:
+        """
+        if not self.ticket_id:
+            logging.error("No ticket ID associated with ticket object. Set ticket ID with set_ticket_id(ticket_id)")
+            return
+
+        # Read the contents from the file path, guess the mimetypes and update the params.
+        f = open(data, "+rb")
```

Comment:
We don't need the '+' in '+rb'. That's only if we wanted to open for writing as well. So, this can be changed to 'rb'.